spec.txt

```diff
@@ -9188,6 +9197,28 @@ foo <a href="\*">
 ````````````````````````````````
 
 
+A block quote can prevent a line from being parsed as inline HTML,
+even though line breaks are allowed in tags:
```

Comment:
Maybe something along the lines of “Block quotes, and other blocks such as headings or lists, precede over inline things:”? My intent here is to show that this isn’t about block quotes per se. Do you think something along those lines makes sense?

Reply:
I think this is a very good point, that in the case of block HTML, as condition 7, line endings are *not* supported, even though the productions of *open tag* and *closing tag* could have them. 🤔

[Using the dingus](https://spec.commonmark.org/dingus/?text=%3Cxxx%0Ayyy%3E):

```markdown
<xxx
yyy>
```

…turns into:

```html
<p><xxx
yyy></p>
```

Importantly, the text for start condition 7 starts with “line begins with” and ends with “followed by the end of the line”, so I think the intent there is to say that it is all on one line. But it is probably good to add that all of this has to be on one line?

Maybe something like this?:

> Start condition: line begins with a complete open tag (with any tag name other than pre, script, style, or textarea) or a complete closing tag, followed by zero or more spaces and tabs, **all on a single line, **followed by the end of the line.

I wonder what @jgm or others think!

Reply:
I wouldn't object to adding "all on a single line" if you think it resolves a doubt or ambiguity.

> Maybe something along the lines of “Block quotes, and other blocks such as headings or lists, precede over inline things:”? My intent here is to show that this isn’t about block quotes per se. Do you think something along those lines makes sense?

See the beginning 3.1 which makes this general assertion about the priority of block-level parsing over inline.